panel/chat/feed.py

```diff
@@ -248,14 +253,17 @@ def __init__(self, *objects, **params):
             visible=self.param.visible
         )
         # we separate out chat log for the auto scroll feature
-        self._chat_log = Column(
+        self._chat_log = Feed(
             *objects,
+            load_buffer=self.load_buffer,
             auto_scroll_limit=self.auto_scroll_limit,
             scroll_button_threshold=self.scroll_button_threshold,
+            view_latest=self.view_latest,
             css_classes=["chat-feed-log"],
             stylesheets=self._stylesheets,
             **linked_params
         )
+        self._chat_log.height = None
```

Comment:
Why is this needed? Should height be removed from `linked_params`?

Reply:
`pn.Feed` sets a fixed height if height is not explicitly set since no local scrollbar appears and thus no buffering is enabled.

Maybe it will be alleviated with this: https://github.com/holoviz/panel/pull/6266